x-pack/plugin/transform/src/test/resources/rest-api-spec/schema/transform_health.schema.json

```diff
@@ -29,9 +29,14 @@
           "description": "describes a single issue",
           "additionalProperties": false,
           "required": [
+            "type",
```

Comment:
Saying `type` is required is incompatible with not including it if it's `null` or an empty string in the `toXContent` method. It's another thing that suggests having a default non-`null`, non-empty value for mixed version clusters.

Reply:
Done.